### DISCUSSION TITLE
Whatshack

### DISCUSSION THREAD

**Discussion by Asabile** [community/community#57685] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to resolve this error
curl: (6) Could not resolve host: apparelworld.org

**Comment by otter-computer** [comment]
Hi there,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
